library/include/G4CMPKaplanQP.hh

```diff
@@ -88,15 +88,18 @@ public:
   void SetLowQPLimit(G4double value)          { lowQPLimit = value; }
   void SetSubgapAbsorption(G4double value)    { subgapAbsorption = value; }
   void SetAbsorberGap(G4double value)         { absorberGap = value; }
+  void SetAbsorberEff(G4double value)         { absorberEff = value; }
+  void SetAbsorberEffSlope(G4double value)    { absorberEffSlope = value; }
   void SetPhononLifetime(G4double value)      { phononLifetime = value; }
   void SetPhononLifetimeSlope(G4double value) { phononLifetimeSlope = value; }
   void SetVSound(G4double value)              { vSound = value; }
 
 protected:
-  // Check that the five required parameters are set to meaningful values
+  // Check that the seven required parameters are set to meaningful values
   G4bool ParamsReady() const {
     return (filmThickness > 0. && gapEnergy >= 0. && vSound > 0. &&
-	    phononLifetime > 0. && phononLifetimeSlope >= 0.);
+	    phononLifetime > 0. && phononLifetimeSlope >= 0. && 
+	    absorberEff > 0. && absorberEffSlope >= 0.);
```

Comment:
Take this (and the comment above) back out.  We want these new parameters to be optional, not required.  That way, the current behaviour remains the default.